appveyor.yml

```diff
@@ -53,6 +53,7 @@ install:
     - cmd: conda config --add channels http://conda.binstar.org/conda-forge
     - cmd: conda info
     - cmd: conda install -n root --quiet --yes conda-build anaconda-client jinja2 setuptools
+    - cmd: if "%TARGET_ARCH%" == "x64" if "%CONDA_PY%" == "34" conda install conda-build=1.20.0 --yes
```

Comment:
@melund I was away, but it seems that @jakirkham fixed that in https://github.com/conda-forge/conda-smithy/pull/143

So in the future we can use `conda-smithy` to update the feedstock, but right now I will merge this as a workaround.

BTW `conda`-smithy is tool to update the feedstocks, lint the recipes, and much more. You can install it with

`conda install -c conda-forge conda-smithy`

and then you can update the feedstock with

`conda smithy rerender`

from the feedstock directory.

Thanks!